hoomd/hpmc/IntegratorHPMC.cc

```diff
@@ -6,6 +6,9 @@
 #include "hoomd/VectorMath.h"
 #include <sstream>
 
+#include <pybind11/stl_bind.h>
+PYBIND11_MAKE_OPAQUE(std::vector<std::shared_ptr<hoomd::hpmc::PairPotential>>);
```

Comment:
I am fairly sure it does. From https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html (in an example on `std::vector<int>`):

> This macro must be specified at the top level (and outside of any namespaces), since it adds a template instantiation of type_caster. If your binding code consists of multiple compilation units, **it must be present in every file** (typically via a common header) preceding any usage of std::vector<int>.

Reply:
Ah thanks for the info. Didn't know that.